<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>fDatosEmpresa</class>
 <widget class="QDialog" name="fDatosEmpresa">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>795</width>
    <height>513</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Datos Empresa</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_29">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_10">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_27">
         <property name="rightMargin">
          <number>9</number>
         </property>
         <item>
          <widget class="QLabel" name="label_10">
           <property name="font">
            <font>
             <pointsize>12</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Datos Fiscales</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Nombre Comercial:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="txtFiscalesNombreComercial"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>* Razón Social:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="txtFiscalesRazonSocial"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>* N.I.F. :</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="txtFiscalesNIF"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_4">
             <item>
              <widget class="QLabel" name="label_4">
               <property name="text">
                <string>Domicilio:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="txtFiscalesDomicilio"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_5">
             <item>
              <widget class="QLabel" name="label_5">
               <property name="text">
                <string>* Código Postal:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="txtFiscalesCodigoPostal"/>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_6">
             <item>
              <widget class="QLabel" name="label_6">
               <property name="text">
                <string>Población:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="txtFiscalesPoblacion"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_7">
             <item>
              <widget class="QLabel" name="label_7">
               <property name="text">
                <string>Provincia:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="txtFiscalesProvincia"/>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_8">
             <item>
              <widget class="QLabel" name="label_8">
               <property name="text">
                <string>Teléfono:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="txtFiscalesTelefono"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_9">
             <item>
              <widget class="QLabel" name="label_9">
               <property name="text">
                <string>Fax:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="txtFiscalesFax"/>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_28">
         <property name="leftMargin">
          <number>18</number>
         </property>
         <item>
          <widget class="QLabel" name="label_11">
           <property name="font">
            <font>
             <pointsize>12</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Datos AUTOESCUELA</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_10">
           <item>
            <widget class="QLabel" name="label_12">
             <property name="text">
              <string>* Nombre Comercial:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="txtAutoNombreComercial"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_11">
             <item>
              <widget class="QLabel" name="label_13">
               <property name="text">
                <string>* Código Provincial Autoescuela:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="txtAutoCodigoProvincial"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_12">
             <item>
              <widget class="QLabel" name="label_14">
               <property name="text">
                <string>* Sección:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="txtAutoSeccion"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_22">
             <item>
              <widget class="QLabel" name="label_24">
               <property name="text">
                <string>* Dígito Control:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="txtAutoDigitoControl"/>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_13">
             <item>
              <widget class="QLabel" name="label_15">
               <property name="text">
                <string>Razón Social:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="txtAutoRazonSocial"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_14">
             <item>
              <widget class="QLabel" name="label_16">
               <property name="text">
                <string>N.I.F. :</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="txtAutoNIF"/>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_6">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_15">
             <item>
              <widget class="QLabel" name="label_17">
               <property name="text">
                <string>Domicilio:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="txtAutoDomicilio"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_16">
             <item>
              <widget class="QLabel" name="label_18">
               <property name="text">
                <string>Código Postal:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="txtAutoCodigoPostal"/>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_7">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_17">
             <item>
              <widget class="QLabel" name="label_19">
               <property name="text">
                <string>Población:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="txtAutoPoblacion"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_18">
             <item>
              <widget class="QLabel" name="label_20">
               <property name="text">
                <string>Provincia:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="txtAutoProvincia"/>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_8">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_19">
             <item>
              <widget class="QLabel" name="label_21">
               <property name="text">
                <string>Teléfono:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="txtAutoTelefono"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_20">
             <item>
              <widget class="QLabel" name="label_22">
               <property name="text">
                <string>E-Mail:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="txtAutoEmail"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_21">
             <item>
              <widget class="QLabel" name="label_23">
               <property name="text">
                <string>Móvil:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="txtAutoMovil"/>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_23">
           <item>
            <widget class="QLabel" name="label_25">
             <property name="text">
              <string>Nombre Titular Autoescuela:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="txtAutoNombreTitular"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_9">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_24">
             <item>
              <widget class="QLabel" name="label_26">
               <property name="text">
                <string>* Nombre Director Autoescuela:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="txtAutoDirector"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_25">
             <item>
              <widget class="QLabel" name="label_27">
               <property name="text">
                <string>N.I.F. : Director:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="txtAutoDirectorNIF"/>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_26">
           <item>
            <widget class="QLabel" name="label_28">
             <property name="text">
              <string>* Localidad de Examen:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="txtAutoLocalidadExamen"/>
           </item>
           <item>
            <spacer name="verticalSpacer_2">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Close|QDialogButtonBox::Save</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>fDatosEmpresa</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>229</x>
     <y>491</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>fDatosEmpresa</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>297</x>
     <y>497</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
